plugins/MauticSocialBundle/Form/Type/FacebookType.php

```diff
@@ -25,7 +26,7 @@ class FacebookType extends AbstractType
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $builder->add('layout', 'choice', [
+        $builder->add('layout', ChoiceType::class, [
             'choices' => [
                 'standard'     => 'mautic.integration.Facebook.share.layout.standard',
```

Comment:
Switch the labels and values for all choice types in this bundle as well. Use the `choices_as_values` option to simulate Symfony 3. See https://docs.google.com/document/d/1lHNK8hHQtvyWlz9OtfAoLupDVb5-QunQjq-HAGlAsS4/edit# for more details.

Reply:
Fixed d9df62781e4c91fa9191c9b2896ff018de22fed2